src/components/Drawer/index.tsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import React from "react";
+
+import Header from "./Header";
+import Owner from "./Owner";
+import Description from "./Description";
+import Team from "./Team";
+import Divider from "../commons/Divider";
+
+export default function Drawer({ isOpen, setIsOpen }: { isOpen: boolean; setIsOpen: any }) {
+  return (
+    <aside
```

Comment:
Try to use our custom `Animate` component instead of hard coding your animation, this will help us write a clean reusable code  that is easier to maintain

Reply:
more explanation i didn't get it

Reply:
Check the visibility drop-down menu in the `Menu`  component, I've used the custom `Animate` component to trigger fade in and fade out animations